tests/test_search_deed.py

```diff
@@ -9,14 +9,14 @@ def setUp(self):
     @with_context
     @with_client
     def test_search_deed_post(self, client):
-        res = client.post('/searchdeed/search', data={'deed_reference': '38'})
+        res = client.post('/searchdeed/search', data={'borrower_token': '38'})
 
         self.assertEqual(res.status_code, 200)
 
     @with_context
     @with_client
     def test_search_deed_post_invalid_reference(self, client):
-        res = client.post('/searchdeed/search', data={'deed_reference': ''})
+        res = client.post('/searchdeed/search', data={'borrower_token': ''})
 
         self.assertEqual(res.status_code, 200)
```

Comment:
The test in question when an invalid code is passed redirects to the not found page. as this is a valid page it would still return a 200 code